match-media.js

```diff
@@ -135,3 +148,60 @@ angular.module('matchMedia', [])
     return that.is(list);
   };
 }]);
+
+// Added my Matthias Max
+// Date: 2015-07-17
+// Version: 1.0
+app.filter('media', ['screenSize', function(screenSize) {
+
+    var mediaFilter = function(inputValue, options) {
+
+      // Get actual size
+      var size = screenSize.get();
+
+      // Variable for the value being return (either a size/rule name or a group name)
+      var returnedName = '';
+
+      if (options) {
```

Comment:
Between this and line#191, could you change this to the following to make it a little less complex:
```javascript
if (!options) {
  return size;
}

if (options.groups) {
  // the rest of the code...
```